code/modules/projectiles/guns/projectile/shotgun.dm

```diff
@@ -215,4 +215,9 @@
 			update_icon()
 		else
 			user << "<span class='warning'>You need at least ten lengths of cable if you want to make a sling.</span>"
-			return
\ No newline at end of file
+			return
+			
+/obj/item/weapon/gun/projectile/revolver/doublebarrel/improvised/sawoff(mob/user as mob)
+	user << "<span class='warning'>Shortening \the [src] will break it.</span>"
+	return

```

Comment:
er; this is supposed to be able to be sawn off; it has its own description for it and a sprite to boot; definitely not a bug either.